account-kit/react/src/components/auth/card/error/types.ts

```diff
@@ -1,4 +1,8 @@
-export type WalletType = "CoinbaseWallet" | "MetaMask" | "WalletConnect";
+export type WalletType =
```

Comment:
This might be a good spot for a string based enum instead.

enum WalletConnectionIdentifier {
   COINBASE_WALLET = "com.coinbase.wallet"
   ....
}

Reply:
Updated!